desk/app/channels.hoon

```diff
@@ -767,11 +767,18 @@
   ::
   ++  ca-has-sub
     ^-  ?
-    (~(has by wex.bowl) [ca-sub-wire ship.nest dap.bowl])
+    (~(has by wex.bowl) [ca-sub-wire ship.nest %channels-server])
   ::
   ++  ca-safe-sub
     |=  delay=?
     ?:  ca-has-sub  ca-core
+    =/  =flag:g  group.perm.perm.channel
+    =/  =path
+      %+  weld
+        /(scot %p our.bowl)/groups/(scot %da now.bowl)
+      /groups/(scot %p p.flag)/[q.flag]/v1/group-ui
+    =+  .^(group=group-ui:g %gx path)
```

Comment:
A little bit surprising that the mark here isn't `%group-ui-v1`, even though the `v0` endpoint produces `%group-ui-v0`...

For added safety you may want to do an `/exists/[flag]` scry first. If the group doesn't even exist, `/groups` scries will just block on you.